py-polars/polars/internals/lazy_functions.py

```diff
@@ -815,14 +816,14 @@ def all(name: Optional[Union[str, tp.List["pli.Expr"]]] = None) -> "pli.Expr":
 
 def groups(column: str) -> "pli.Expr":
     """
-    Syntactic sugar for `column("foo").agg_groups()`.
+    Syntactic sugar for `pl.col("foo").agg_groups()`.
```

Comment:
Upstream this should already be the case.
```✦ ❯ rg -A 10 'def groups'
polars/internals/lazy_functions.py
824:def groups(column: str) -> "pli.Expr":
825-    """
826-    Syntactic sugar for `pl.col("foo").agg_groups()`.
827-    """
828-    return col(column).agg_groups()
829-
830-
831-def quantile(column: str, quantile: float) -> "pli.Expr":
832-    """
833-    Syntactic sugar for `pl.col("foo").quantile(..)`.
834-    """
```

I have the feeling it would be easier to start from scrach again as a lot of changes are already done.

Reply:
You can try the following (and fix merge conflicts).

```
git checkout master
git pulll
git checkout fix_multiline_docstring
git rebase master
```

Reply:
Or start a new branch and only cherry-pick the commits that are still relevant.

```
git checkout master
git pull
git branch fix_docstring
git checkout fix_docstring
git cherry-pick e4e6f2f2a9d63531ec03f613da2e24cadde6ad5a
# Fix commit if necessary.
```

Reply:
You are responding to outdated changes, i.e. before I rebased. Please respond on the latest version.

Reply:
The commits here confused me.

I now looked at: https://github.com/pola-rs/polars/pull/1935/files and it looks OK now :-)